python-api-examples/offline-omnilingual-asr-ctc-decode-files.py

```
@@ -0,0 +1,133 @@
+    test_wav_fr = "./sherpa-onnx-omnilingual-asr-1600-languages-300M-ctc-int8-2025-11-12/test_wavs/fr.wav"
+    test_wav_es = "./sherpa-onnx-omnilingual-asr-1600-languages-300M-ctc-int8-2025-11-12/test_wavs/es.wav"
+
+    for f in [model, tokens, test_wav_en, test_wav_de, test_wav_fr, test_wav_es]:
+        if not Path(f).is_file():
+            print(f"{f} does not exist")
+
+            raise ValueError(
+                """Please download model files from
+                https://github.com/k2-fsa/sherpa-onnx/releases/tag/asr-models
+                """
+            )
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)